app/models/user.rb

```diff
@@ -48,6 +46,10 @@ def github_account?(github_account)
     end
   end
 
+  def slack_teams
```

Comment:
A better way to do this is to use [`has_many :through`](http://guides.rubyonrails.org/association_basics.html#the-has-many-through-association):

```ruby
has_many :slack_teams, through: :slack_accounts
```

This'll allow you to use eager loading on `user.slack_teams`.